docs/howto/build-run.md

```diff
@@ -152,6 +152,37 @@ Apart from the steps mentioned below, you may find the
 
 [React Native troubleshooting docs]: https://reactnative.dev/docs/troubleshooting
 
+### `tools\postinstall` is not recognized as an internal or external command
+
+On windowsOS, when running `yarn` or `yarn install`, if you see an error like
+this:
+```
+yarn install
+yarn install v1.22.5
+[1/4] Resolving packages...
+success Already up-to-date.
+$ tools/postinstall
+'tools\postinstall' is not recognized as an internal or external command,
+operable program or batch file.
+error Command failed with exit code 1.
+info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
+```
+then, in our `Package.json` :
```

Comment:
Do you literally have a `Package.json` file, with a capital `P`? Or do you mean `package.json`?

Reply:
Just a typo, fixing ...